### PR TITLE
fix a function name and variable package collide

### LINKED ISSUE #1009: [BUG]fix a function name and variable package collide
**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

### PR DESCRIPTION
**Pull request by kitianFresh**
### Ⅰ. Describe what this PR does
fix a function name and variable name collide with package name

### Ⅱ. Does this pull request fix one issue?

fixes #1009 

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
NO，just rename some function name

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by kitianFresh** [comment]
Hi team, thanks for reviewing this. @ChenYi015

File: pkg/controllers/v1alpha1/dataload/dataload_controller.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (r *DataLoadReconciler) AddOwnerAndRequeue(ctx cruntime.ReconcileRequestCon
 	return utils.RequeueImmediately()
 }
 
-func (r *DataLoadReconciler) addFinalierAndRequeue(ctx cruntime.ReconcileRequestContext, targetDataload datav1alpha1.DataLoad) (ctrl.Result, error) {
+func (r *DataLoadReconciler) addFinalizerAndRequeue(ctx cruntime.ReconcileRequestContext, targetDataload datav1alpha1.DataLoad) (ctrl.Result, error) {
```
Comment (cheyang) [inline pkg/controllers/v1alpha1/dataload/dataload_controller.go:192]: Thank you for fixing it!

File: pkg/controllers/v1alpha1/dataload/dataload_controller.go [hunk 2]
```diff
@@ -120,14 +120,14 @@ func (r *DataLoadReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 	ctx.RuntimeType = boundedRuntime.Type
 
-	var runtime runtime.Object
+	var fluidRuntime runtime.Object
```
Comment (cheyang) [inline pkg/controllers/v1alpha1/dataload/dataload_controller.go:123]: May I know the reason of changing this?

Comment (kitianFresh) [reply pkg/controllers/v1alpha1/dataload/dataload_controller.go:123]: > May I know the reason of changing this?

the variable name runtime collide  with the import package name runtime

Comment (cheyang) [reply pkg/controllers/v1alpha1/dataload/dataload_controller.go:123]: Does it matter in compiling or runtime?

Comment (kitianFresh) [reply pkg/controllers/v1alpha1/dataload/dataload_controller.go:123]: Now it does not affect, I just want to use a more elegant naming. I can rollback it to the original name if you think so

Comment (cheyang) [reply pkg/controllers/v1alpha1/dataload/dataload_controller.go:123]: No problem. I'm just curious about the reason.

**Review by cheyang — APPROVED** [review]

**Pull request merged**